frontend/packages/dev-console/integration-tests/features/monitoring/monitoring-alerts.feature

```diff
@@ -0,0 +1,155 @@
+Feature: Filter for Alert state and Severity
+	User should be able to filter out alerts as per Alert State and Severity
+
+
+    Background:
+        Given user is at developer perspective
+        And user is at Add page 
+        And user has created or selected namespace "aut-monitoring-alerts"
+        # To configure the alerts on cluster - need to execute below yaml files from monitoring YAMLs test data
+        # cluster-monitoring-config.yaml, workload-monitoring-config.yaml, prometheous-example.yaml
+        And user is at Monitoring page
+
+
+    @smoke
+    Scenario: Filters dropdown on the Alerts page
+        Given user is on the Alerts tab
+        When user clicks on the Filter dropdown
+        Then user is able to see Firing, Pending, Silenced and Not Firing filters under Alert State type
+        And user is able to see Critical, Warning, Info and None filters under Severity type
+        And user is able to see filters as unchecked
+
+
+    @regression, @manual
+    Scenario Outline: Ability to show only "<alert_state>" Alerts on the Alerts tab
+        Given user is on the Alerts tab
+        When user selects the "<alert_state>" option under Alert State type
+        Then user will see the only "<alert_state>" alerts if there are any
+
+    Examples:
+        | alert_state |
+        | Firing      |
+        | Pending     |
+        | Silenced    |
+        | Not Firing  |
+
+
+    @regression, @manual
+    Scenario Outline: Ability to show only "<severity>" Alerts on the Alerts tab
+        Given user is on the Alerts tab
+        When user selects the "<severity>" option under Severity type
+        Then user will see the only "<severity>" alerts if there are any
+
+    Examples:
+        | severity |
+        | Critical |
+        | Warning  |
+        | Info     |
+        | None     |
+
+
+    @regression, @manual
+    Scenario Outline: Ability to hide only "<alert_state>" Alerts on the Alerts tab
+        Given user is on the Alerts tab
+        When user selects the "<alert_state>" option under Alert State type
+        Then user will not see the "<alert_state>" alerts
+
+    Examples:
+        | alert_state |
+        | Firing      |
+        | Pending     |
+        | Silenced    |
+        | Not Firing  |
+
+
+    @regression, @manual
+    Scenario Outline: Ability to hide only "<severity>" Alerts on the Alerts tab
+        Given user is on the Alerts tab
+        When user selects the "<severity>" option under Severity type
+        Then user will not see the "<severity>" alerts
+
+    Examples:
+        | severity |
+        | Critical |
+        | Warning  |
+        | Info     |
+        | None     |
+
+
+    @regression
+    Scenario: Alert details page
+        Given user is on Alerts tab
+        When user clicks on the name of the alert
+        Then user will see Alert details page
+        And user will see alert Metrics
+        And user will see Name, Severity, Labels, Source, State, Alerting rule
```

Comment:
Isn't this regression is similar to this
https://github.com/openshift/console/pull/8132/files#diff-300a611e03ff2b74764a0d4d1c10ee3e6dc7e0726afb299ab5cd2605e3c686fbR152?

Reply:
No, this is from the Alert tab in the monitoring and that one you mentioned is from the Alert rule details page

Reply:
ah got it. Thanks